From d2259556389e1e1279bbb9f6fa2f4c2801516656 Mon Sep 17 00:00:00 2001
From: Gareth Williams <gareth.williams.jx@renesas.com>
Date: Wed, 9 Mar 2022 11:58:15 +0000
Subject: [PATCH] tensorflow: lite: Do not build XNNPack into library output

For benchmarking purposes we need to be able to change the delegate
at runtime however there is still a need to be able to enable and
link against the XNNPack library. For this we build XNNPack but do not
add it to libtensorflow-lite.a or enable it by default.

Signed-off-by: Gareth Williams <gareth.williams.jx@renesas.com>
---
 tensorflow/lite/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tensorflow/lite/CMakeLists.txt b/tensorflow/lite/CMakeLists.txt
index 74f1dc8..820e802 100644
--- a/tensorflow/lite/CMakeLists.txt
+++ b/tensorflow/lite/CMakeLists.txt
@@ -307,7 +307,6 @@ if(TFLITE_ENABLE_XNNPACK)
   list(APPEND TFLITE_TARGET_DEPENDENCIES
     XNNPACK
   )
-  list(APPEND TFLITE_TARGET_PUBLIC_OPTIONS "-DTFLITE_BUILD_WITH_XNNPACK_DELEGATE")
 endif()
 if (TFLITE_ENABLE_RESOURCE)
   populate_tflite_source_vars("experimental/resource"
-- 
2.7.4

